main/common/resources/resources.js

```diff
@@ -179,12 +185,7 @@
       'search': {
         isArray: true,
         url: API_PATH + 'bpm/processConnector/',
-        interceptor: {
-          response: function(response) {
-            response.resource.pagination = parseContentRange(response.headers('Content-Range'));
-            return response;
-          }
-        }
+        interceptor: contentRangeInterceptor
```

Comment:
it would be nice to factorize the variable contentRangeInterceptor to the global factory service creator (it is in the same file)